pep-0458.txt

```diff
@@ -263,9 +263,9 @@ example, TUF MAY be integrated with the pip package manager. Thus, new versions
 of pip going forward SHOULD use TUF by default to download and verify packages
 from PyPI before installing them. However, there may be unforeseen issues that
 might prevent users from installing or updating packages, including pip itself,
-via TUF. Therefore, pip MAY provide an undocumented option
-(e.g., `--no-pep-458`) in order to work around such issues until they are
-resolved.
+via TUF. Therefore, pip SHOULD provide an option (e.g.,
+`--unsafely-disable-package-verification`) in order to work around such issues
```

Comment:
I deliberated on this for a bit and ultimately decided that wordy and explicit was better - the action is unsafe and not generally recommended, it disables package verification.

I'm not sure temporary is the right word, the option would disable package verification for that invocation of pip but the unverified packages that are installed will persist on the system permanently (until updated or removed).

Thinking out loud here, does it also make sense to suggest an option for pip to undo unsafe usage, i.e. re-download and verify all installed packages? AFAIK pip doesn't store packages once they are installed, so this would require re-fetching any installed packages.

Reply:
I agree with you that clarity is probably the most important thing in labeling this option, so as ungainly as the phrase seems, you are probably correct in going forward with what you have.

Reply:
I am erring on the side of "probably not worth the effort" too - I anticipate it would require significant changes to Pip and the outcome can be achieved in different ways (i.e. venv and nuke/replace that if you've had to use `--unsafely-disable-package-verification`)

Reply:
Besides `--disable-*`, pip also uses `--force-*`, which might make a more serious impression, e.g.  `--force-unsafe-install`.